dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeClient.java

```diff
@@ -39,32 +40,27 @@
 
     private final Client client;
     private final ExchangeChannel channel;
-    // heartbeat(ms), default value is 0 , won't execute a heartbeat.
     private int heartbeat;
-    private int heartbeatTimeout;
+    private int idleTimeout;
 
-    private HashedWheelTimer heartbeatTimer;
+    private static HashedWheelTimer idleCheckTimer = new HashedWheelTimer(new NamedThreadFactory("dubbo-client-idleCheck", true), 1,

```

Comment:
I think we should establish a mapping relationship between client and task.

Since we are using the global Timer at the moment, when any client is closed, we should cancel the task corresponding to this client. In this way, we can guarantee that our tasks will not be too much.